test/algorithms/buffer/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@
 foreach(item IN ITEMS
     buffer
     buffer_gc
+    buffer_variable_width
```

Comment:
this is the case of last PR, but was not in the cmake yet because that PR was made before